boxsdk/object/upload_session.py

```diff
@@ -18,6 +19,7 @@ class UploadSession(BaseObject):
     _item_type = 'upload_session'
     _parent_item_type = 'file'
 
+    @api_call
```

Comment:
Is there any reason for adding api_call decorator to get_url() method? In other places we didn't put this decorator on this method

Reply:
looks like this doesn't result in a network call so I removed it.